packages/truffle/scripts/webpack5hack.js

```diff
@@ -0,0 +1,32 @@
+const fs = require("fs");
```

Comment:
Can we change the name of this file to something semantic, like `patchHighlightJsSolidity.js`?

Reply:
Absolutely

Reply:
Fixed and committed. Marked as resolved.